### ISSUE TITLE
Main Template breaks Signal calls

### ISSUE THREAD

**Issue by m3thm4th** [KodoPengin/GameIndustry-hosts-Template#22]
The Main Template list blocks [Signal](https://signal.org) calls, I tried to diagnose the issue and found on the list only one domain which is used by Signal calls, that is `stun1.l.google.com`. Now, that domain is commented with a `#`, I assume AdAway is not picking up the fact that it is commented, but after all there's no need to keep all of those commented domains, they make the list bigger while serving no real purpose. In fact I remember using this list a few years ago and it broke Android functionalities, which I tracked down to a commented domain.

**Comment by KodoPengin** [comment]
Yes, they have unlocked during the last tests and updates. Was also a request which can be seen in the past issues.

Well, the whole hosts structure is aimed at showing people what is included in each product and where product XY (game, program, website, whatever) connects to. Including commented entries. This can also be seen on the [website](https://gameindustry.eu/en/search/)

Therefore, the entries remain. Feel free to delete related entries for your own purpose.

They do not really take up space. Compared to other lists that contain hundreds of thousands to millions of entries, my list is vanishingly small and i would say better documented (I know, unfortunately not in english) and sorted.

To be honest, if an addon or program has problems with commented entries, would it be time to look for an alternative?

---

With Android, i am thinking about to rebuild and merge my files. The last updates have been a bit longer by now.

**Comment by KodoPengin** [comment]
I'll try to check Signal during the weekend,

**Comment by KodoPengin** [comment]
Since there was no further feedback anymore, the issue will be closed

**Comment by KodoPengin** [comment]
No worry. Just looked today again to see if there are new entries because I wanted to publish a new update as soon as possible.

I have now tested Signal again.
Latest version 6.13.8 from the first link, direct APK download from the Signal website.
Active Lists used by me > Android-Full-Template, Android-Mini-Template, Main-Template

Setup 1 with Pcapdroid to see connections to verify connections
Setup 2 all traffic redirected through PI-Hole to verify connections

Sorry, but I still can't verify the problem. Should this change, the lists will of course be updated accordingly.
